### PR TITLE
Updated readme to explain jumping to line numbers.

### PR DESCRIPTION
**Pull request by calebmeyer**
If we'd like more explanation, I can add it. If we want another photo, I'll have to figure out how to get it on f.cloud.github.com

### CODE REVIEW

**Comment by lee-dohm** [comment]
If you can split out the <kbd>Enter</kbd>, <kbd>Shift+Enter</kbd> and line number stuff into a separate list, I think it would be more readable.

File: README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Quickly find and open files using `cmd-t`.
   * `cmd-shift-b` to open the list of Git modified and untracked files
   * `enter` defaults to opening the selected file without leaving the current pane
   * `shift-enter` defaults to switching to another pane if the file is already open there
+  * `:<line number>` will go directly to the line number you specify, or the last line if the number is larger
```
Comment (lee-dohm) [inline README.md]: Rewriting the beginning of this to:

> Adding `:<line number>` to the end of your search text will go directly ...

I think will be more clear.

**Comment by biesmard** [comment]
I am the right person to develop

**Comment by smashwilson** [comment]
:+1: Thanks!

**Pull request merged**